x/tradeshield/keeper/pending_spot_order.go

```diff
@@ -209,9 +208,9 @@ func (k Keeper) ExecuteStopLossOrder(ctx sdk.Context, order types.SpotOrder) (*a
 		Sender:    order.OwnerAddress,
 		Recipient: order.OwnerAddress,
 		Amount:    order.OrderAmount,
-		DenomIn:   order.OrderPrice.BaseDenom,
-		DenomOut:  order.OrderPrice.QuoteDenom,
-		MinAmount: sdk.NewCoin(order.OrderTargetDenom, sdkmath.ZeroInt()),
+		DenomIn:   order.OrderPrice.QuoteDenom,
+		DenomOut:  order.OrderPrice.BaseDenom,
+		MaxAmount: order.OrderAmount,
```

Comment:
this was fine earlier, I think

Reply:
SL and sell order are pretty much the same flow, that’s why I thought it would also make sense to do the change for SL

Reply:
sell order if fine, the issue is in limit_buy only, market_buy is fine too